doc/holochain_101/src/zome/api_functions.md

```diff
@@ -40,48 +40,52 @@ zome logic to Holochain core functionality and often has side effects. The
 return value of the Rust function is passed back to the zome code as the return
 of the zome API function.
 
-## Reference
-
-Note: Full reference is available in language-specific API Reference documentation.
-(TODO add links)
-
 ### Property
 
 Canonical name: `property`
 
 Returns an application property, which are defined by the app developer in the DNA.
 It returns values from the DNA file that you set as properties of your application (e.g. Name, Language, Description, Author, etc.).
 
+[LINK](https://holochain.github.io/rust-api/hdk/fn.property.html)
+
 ### Make Hash
 
 Canonical name: `make_hash`
 
-TODO
+Not yet available, but you will see
```

Comment:
This one is available and has been renamed to `hash_entry`. Could you make a note of that here? (and rename the heading too?)